src/reflect/type.go

```diff
@@ -1171,6 +1171,7 @@ var ptrMap sync.Map // map[*rtype]*ptrType
 //
 // PtrTo is the old spelling of PointerTo.
 // The two functions behave identically.
+// Deprecated: Superseded by [PointerTo]
```

Comment:
I think there must be an empty comment line before the Deprecated line, otherwise it may not be picked up as "deprecated"